R/spark_compile.R

```diff
@@ -126,7 +126,7 @@ spark_compile <- function(name,
 #'
 #' @keywords internal
 #' @export
-spark_compile_package_jars <- function(package = rprojroot::find_package_root_file(),
+compile_package_jars <- function(package = rprojroot::find_package_root_file(),
                                        spark_versions = NULL,
```

Comment:
The tough part about scala_versions is that we don't really have a good way of indexing the `scala` compilers. Maybe we need an entry like:

```
scala_compilers = list(
    "2.10" = "path/to/scala-2.10",
    "2.11" = "path/to/scala-2.11"
)
```

or something to that effect?